### PR TITLE
Enabling the Tile Component to utilise the image component

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do?
- Adjust the tile component to utilise the image component
- Updates the docs
- Adds some default styling for images
- Spaces out the articles on docs better

#### What unit or integration tests does this PR have?
None

#### What selenium tests does this PR have?
None

#### How should a developer review this?
The best way to test this would be to `console.log` in the image component and make sure the log is shown when running the docs

#### How should this be manually tested?
`npm install`
`grunt docs`
Check that images are displayed on the docs

#### Any background context you want to provide?
- The image component was produced but the tile needed to be updated.

#### What are the relevant tickets?
UXUI-250

#### What gif best describes how you feel about this work?
drunken monkey
---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [x] I've checked there is appropriate unit, selenium and regression test coverage.
- [x] I've updated documentation with any changes to procedures.
- [x] I've tested this cross-browser/device for visual changes.
- [x] I've checked this work against the requirements of the Jira.
- [x] I've manually checked this work against any dependent systems.
- [x] I've added and updated translations for all supported languages.
- [x] I've informed those who need to know of any unforeseen impact of this work (database migrations, deployment procedure changes).
- [x] I've checked that there are no negative speed or performance impacts from my work.
- [x] I've added appropriate tracking.
- [x] I am ready for this to be code reviewed, merged and tested.

#### IP or Developer Review:
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve run all the tests locally and they pass.
- [x] +1 from me!

#### Software Engineer or Developer review:
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve run all the tests locally and they pass.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] +1 from me!

#### Software Engineer or project guru review:
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve run all the tests locally and they pass.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

### CODE REVIEW

**Comment by LouisPryer** [comment]
Taking IP/Dev

File: docs/examples/Tile.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 var image = {
   src: 'http://brand.holidayextras.com/img/product-main.jpg',
-  alt: 'Two course meal'
+  alt: 'Tender sirloin steak with blue cheese dressing'
```
Comment (LouisPryer) [inline docs/examples/Tile.jsx]: Should we change `<UIToolkit.Tile image={image} title` on line 7, to also equal this?

**Comment by matthewjtyas** [comment]
Taking SE/Dev

**Comment by matthewjtyas** [comment]
Looks good to me +1

**Comment by lukehansell-hx** [comment]
taking guru

**Comment by lukehansell-hx** [comment]
Can we start adding some real tests for these components now? Perhaps one that asserts the imageComponent is rendered and is in the correct place would be a good start?

**Comment by LouisPryer** [comment]
:+1:

**Comment by jodiedoubleday** [comment]
@lukehansell-hx There is a PR to move from JEST to Mocha/Chai so I don't want to write tests here which conflict. More tests will be written once we have moved.

**Comment by lukehansell-hx** [comment]
:+1: from me...

can I ask what the change in test suite is for? I'm not overly fond of Jest, but I do like the stubbing functionality it gives us.

**Comment by jodiedoubleday** [comment]
@lukehansell-hx we have moved for a few reasons..
- We couldn't get coverage reports for Jest (JSX issues)
- Jest doesn't support Node 12

**Pull request merged**